src/ram/MergeExtend.h

```diff
@@ -48,15 +48,15 @@ class Extend : public BinRelationStatement {
         return getSecondRelation();
     }
 
-    Extend* cloning() const override {
-        auto* res = new Extend(second, first);
+    MergeExtend* cloning() const override {
+        auto* res = new MergeExtend(second, first);
         return res;
     }
 
 protected:
     void print(std::ostream& os, int tabpos) const override {
         os << times(" ", tabpos);
-        os << "EXTEND " << getTargetRelation() << " WITH " << getSourceRelation();
+        os << "MERGE EXTEND " << getTargetRelation() << " WITH " << getSourceRelation();
```

Comment:
Can we print "MERGE-EXTEND" with a hyphenation? In future, we may want to have a parser for RAM and we would like to have only a single keyword for the operation.